chart/values.yaml

```diff
@@ -13,7 +13,6 @@ eo-base-helm-chart:
   fullnameOverride: "eo-auth"
 
   env:
-    TOKEN_COOKIE_SAMESITE: true
```

Comment:
Already specified default at https://github.com/Energinet-DataHub/eo-auth/blob/009071018a390aeee29f2ab0da472b1338ea9f89/src/auth_api/config.py#L37

This has to be removed since it takes precedens over the configmap.